src/Resistor/ResistorColorEntry.ts

```diff
@@ -0,0 +1,31 @@
+class ResistorColorEntry {
+  public readonly name: string;
+  public readonly colorCode: string;
+  public readonly multiplier: number;
+  public readonly value?: number;
+  public readonly tolerance?: string;
+
+  constructor(name: string, colorCode: string, multiplier: number, value?: number, tolerance?: string) {
+    this.name = name;
+    this.value = value;
+    this.colorCode = colorCode;
+    this.multiplier = multiplier;
+    this.tolerance = tolerance;
+  }
+
+  public getDisplayValue() {
+    if (this.value === undefined) {
+      return '';
+    }
+    return String(this.value);
+  }
+
+  public getDisplayTolerance() {
+    if (this.tolerance === undefined) {
+      return '';
+    }
+    return String(this.tolerance);

```

Comment:
Isn't this already a string anyway?  If you do take my recommendation to have this be a number you should be able to just do `` `${this.tolerance}%` ``

Reply:
......yeah, as I was implementing your previous comment I realized what I had done there.